ddocs/medic-db/medic-client/views/doc_by_type/map.js

```diff
@@ -6,5 +6,12 @@ function(doc) {
     });
     return;
   }
+  if (doc.type === 'user-settings') {
```

Comment:
I'm a little worried by this change. This will mean that all clients (users' phones) will need to index this massive view again due to this change that only gets used on the server side.
I find it hard to see a context where we would need this change on the client. I think it makes more sense to put this new view code in `_design/medic` somehow.

Reply:
Good catch there. I don't see any existing view in `_design/medic` we could piggyback off of so we'd have to make a new view.
The way I see it, I could either duplicate the `medic-client/doc_by_type` code to a `medic/doc_by_type` view and pop the changes there (and potentially redirect all the `medic-client/doc_by_type` queries in API to this view) or make a specific view named like `medic/user_settings_by_facility_and_contact` that would look like this:
```ts
function(doc) {
  if (doc.type === 'user-settings') {
    emit(doc.facility_id);
    emit(doc.contact_id);
    emit([doc.facility_id, doc.contact_id]);
  }
}
```

I'm leaning towards the first option and duplicate the view and re-map API queries from the client view to have a distinct API-only `doc_by_type` view so that we can make changes to it without forcing clients to index it again.

Reply:
IMHO, we probably only want to do the first option if it would mean that we could avoid having to build the `medic-client/doc_by_type` on the server at all.  Otherwise, having both `medic-client/doc_by_type` and `medic/doc_by_type` would mean a non-trivial addition to the disk space being used on the server.  Do we support having views on the server _that do not get warmed?_